### PR TITLE
EntityStore: Rename ObjectStore to EntityStore (part 1)

### PR DESCRIPTION
**Pull request by ryantxu**
See https://github.com/grafana/grafana/issues/59523

This PR is just renaming packages, structs, and protobuf -- there are no functional changes

### CODE REVIEW

File: pkg/api/api.go [hunk 1]
```diff
@@ -288,7 +288,7 @@ func (hs *HTTPServer) registerRoutes() {
 
 			// Allow HTTP access to the object storage feature (dev only for now)
 			if hs.Features.IsEnabled(featuremgmt.FlagGrpcServer) {
-				apiRoute.Group("/object", hs.httpObjectStore.RegisterHTTPRoutes)
+				apiRoute.Group("/object", hs.httpEntityStore.RegisterHTTPRoutes)
```
Comment (toddtreece) [inline pkg/api/api.go:291]: should this be `/entity`?

**Review by toddtreece — APPROVED** [review]

File: pkg/api/api.go (hunk 1 above)
Comment (ryantxu) [reply pkg/api/api.go:291]: yup -- I am sure there are more 😬

Comment (ryantxu) [reply pkg/api/api.go:291]: I will merge this as is, and create a new PR that has a few more changes with less automatic renamings

Comment (toddtreece) [reply pkg/api/api.go:291]: good call. it's pretty hard to spot in this one

**Pull request merged**